webapp/channels/src/components/dot_menu/post_reminder_submenu.tsx

```diff
@@ -141,9 +141,12 @@ function PostReminderSubmenu(props: Props) {
         );
     });
 
+    const header = formatMessage({id: 'post_info.post_reminder.sub_menu.header', defaultMessage: 'Set a reminder for:'});
+
     return (
         <Menu.SubMenu
             id={`remind_post_${props.post.id}`}
+            menuAriaLabel={header.replace(':', '')}
```

Comment:
Replacing on a translated string is dangerous. Hopefully the colon is the same in all languages (I check the languages where that string is translated and seems fine).

Reply:
And also as Harrison commented out in my closed PR, `replace` would be good to move up where `header` is declared.

Reply:
I think I'll just remove the replace and have the E2E test look for it. I don't think a screen reader should read out the colon, so I think that'd be fine. Does that sound like it'd work at least from the test side, @saturninoabril?

Reply:
I originally thought of omitting `:` primarily due to screen reader but I just checked and you're correct that it's not being read out. Sounds good to remove the replace.

Reply:
Will this be resolved in this PR?

Reply:
It should've been, but I forgot to push my "address feedback" commit when I fixed it last week 😅